### ISSUE TITLE
winbuild: Schannel is documented as true by default; but is actually false

### ISSUE THREAD

**Issue by ganeshkamath89** [curl/curl#6216]
The Schannel property is documented as true by default in the document: curl\winbuild\Makefile.vc

I noticed during the build that it prints
_1>Using NGHTTP2: false
1>Using c-ares:
1>Using SSH2:
1>Using ZLIB:
1>Using IDN: true
1>Using IPv6: true
1>Using SSPI: true
1>Using Schannel: false_

So I explicitly added the ENABLE_SCHANNEL=yes

Now I am getting the desired result.

**Comment by bagder** [comment]
I looked at the Makefile but I couldn't figure out why this happens (and I don't have a Windows system around to debug it on).

**Comment by jay** [comment]
@ganeshkamath89 please give the full command line. If you overrode the SSL library or turned off SSPI then schannel is disabled by default.

**Comment by ganeshkamath89** [comment]
@jay , here is the full command line
`nmake /f Makefile.vc mode=dll debug=yes WITH_SSL=dll ENABLE_IPV6=yes`

Thank you for the clarification. I think this case too can be documented in the same file.

**Comment by bagder** [comment]
Setting `WITH_SSL` means that you select to use OpenSSL which is documented (which then makes Schannel default to false, which is implied and maybe not so clear).

**Comment by bagder** [comment]
Still, I don't think this is a bug as stated, since this was not done with the defaults.

**Comment by jay** [comment]
I agree it's not clear. Maybe we should change the readme to say defaults to yes if no other SSL library is specified? Or maybe we should change the makefile to enable it by default regardless?

for example
~~~
 - `ENABLE_SCHANNEL=<yes/no>`    - Enable native Windows SSL support, defaults
                                   to yes if SSPI and no other SSL library
~~~

/cc @MarcelRaad @vszakats

**Comment by ganeshkamath89** [comment]
@jay In my opinion it is best to just update the documentation and keep the functionality the way it is. Few benefits:
1) It will not require any change for existing users.
2) Adding SCHANNEL support explicitly helps make better decision than assuming it is on.

When I enabled SCHANNEL explicitly, TLS1.3 worked on Windows Insider build but not on regular build.

Only after disabling again I came to know that I had to specify TLS backend to openssl in code, for it to work with TLS 1.3.
